app/views/admin/reporting/service_surveys.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :navigation_title do %>
+  Service Surveys by Agency
+<% end %>
+<div class="agency-service-survey-rerport">
```

Comment:
any way to cleanup this markup?

I'm using Atom and HTML beautify is a useful helper.
Maybe there's something to align the HTML?